opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableEditionContainer.jsx

```diff
@@ -49,7 +49,7 @@ const StixCyberObservableEditionContainer = (props) => {
         >
           <Close fontSize="small" color="primary" />
         </IconButton>
-        <Typography variant="h6" classes={{ root: classes.title }}>
+        <Typography variant="subtitle2" classes={{ root: classes.title }}>
```

Comment:
Ok, your fix might work but I think there is a better way : 

We do have an internal Drawer component that is using MaterialUI inside. 
I think the correct fix is to handle the usages of StixCyberObservableEditionContainer by replacing the old Drawer by the new one. (For example you can look at ReportCreation.tsx to see how we use the new Drawer)